src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTickets {
+    private final List<LottoNumbers> lottoTickets;
+
+    public LottoTickets(final List<LottoNumbers> lottoTickets) {
+        checkNullOrEmpty(lottoTickets);
+        this.lottoTickets = lottoTickets;
+    }
+
+    private void checkNullOrEmpty(final List<LottoNumbers> lottoTickets) {
+        if (Objects.isNull(lottoTickets) || lottoTickets.isEmpty()) {
```

Comment:
티켓에 대한 유효검증 잘해주셨습니다!

한가지 전달 드리고 싶은 부분은 아래와 같이 lottoTickets.isEmpty()만 조건에 넣으셔도 될 것 같습니다.

이유는 Objects.isNull(lottoTicktes) <- 겉으로 보기엔 null체크하는 코드로 보이지만 Objects.isNull메서드를 확인해 보시면 받아온 값이 null인지를 판단하는 로직이 있는데요. list는 null이 아니고 size가 0이기 때문에 비어 있는 경우면 통과됩니다. :-) 
참고 부탁드려용! 
```suggestion
        if ( lottoTickets.isEmpty()) {
                   throw new IllegalArgumentException("lottoTickets는 null이거나 empty 일 수 없습니다."); 
        }
```

Reply:
리뷰어님 답변에 감사드립니다.

그런데 리뷰어님 코멘트대로 작성했을 때 만약에 생성자로 null이 들어오면 NullPointException이 발생하지 않을까요?

LottoTickets lottoTickets = new LottoTickets(null);

이런 식으로요!

Reply:
넵, 말씀주신것처럼 그런경우는 null이 들어오게 되는게맞지만 
지금 코드는 list를 검증하는 부분이라 위에 처럼 코멘트 드린것입니다 
리스트에 값을 넣을때 null인 객체는 안넣지 않을까싶습니다 :-)